SPECS/kernel/kernel.spec

```diff
@@ -572,6 +575,10 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 
 
 %changelog
+* Mon Jan 3 2022 Cameron Baird <cameronbaird@microsoft.com> - 5.10.88.1-1
+- Update Kernel source to 5.10.88.1
+- Fixes CVE-43975
```

Comment:
Please mention both CVEs and create a new line for the patch
- Addressed CVE-2021-43975
- Patch for CVE-2021-43976